channels/iq.m3u

```diff
@@ -105,6 +107,10 @@ https://5aa8c7549bfa9.streamlock.net/liveTrans/ngrp:SHOW1_all/playlist.m3u8
 http://176.10.117.18:8000/play/a006/index.m3u8
 #EXTINF:-1 tvg-id="KurdSatHD.iq" tvg-name="KurdSat HD" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://kurdmaxcms.com/asset/image/footer_logo/show_2.png" group-title="",KurdSat HD
 http://ikomg2.s.llnwi.net/kurdsathd/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="IQ" tvg-language="Kurdish" tvg-logo="https://i.imgur.com/c5ACcvv.jpg" group-title="News",Kurdsatnews
+https://ikomg2.mmdlive.lldns.net/ikomg2/107b7df8f5444d778f349100739a09cd/manifest.m3u8
```

Comment:
alternative link would be http://ikomg2.s.llnwi.net/kurdsatnewshd/playlist.m3u8 if you want to add it.